common/config/azure-pipelines/templates/publish.yml

```diff
@@ -11,7 +11,24 @@ steps:
   - script: node common/scripts/install-run-rush.js rebuild -v
     displayName: rush rebuild
 
-  - script: node common/scripts/install-run-rush.js publish --publish --include-all --set-access-level public
+  - script: git config --local user.email imodeljs-admin@users.noreply.github.com
+    displayName: Set GitHub user email (needed by `rush version`)
+
+  - script: node common/scripts/install-run-rush.js version --ensure-version-policy
+    displayName: Ensure consistent version across all packages
+    
+  - powershell: |
```

Comment:
im not sure but i think this might be run too early,
i believe you want to check the version after the version bump which occurs in the publish step (i think)
might want to double check this

Reply:
publish step does not bump the version, we still bump it manually. the publish step just publishes whatever is built to npm